examples/example_loadfile.m

```diff
@@ -0,0 +1,10 @@
+% Initialization.
+clear;
+setup;
+n = neuron.Neuron();
+
+% Load stdrun.hoc; run n.continuerun
+n.t = 0;
+n.load_file('stdrun.hoc');
```

Comment:
Ok to do relative file reference, or should construct full filename?

Reply:
I think in principle `stdrun.hoc` can always be found this way, because if you run an example you are either in the `examples` directory or that directory is on your path.